requirements_test.txt

```diff
@@ -48,7 +48,6 @@ types-pkg-resources==0.1.3
 types-psutil==5.9.5
 types-python-dateutil==2.8.19.13
 types-python-slugify==0.1.2
-types-pytz==2023.3.0.0
```

Comment:
You'll likely have to leave this in though for the deps

Reply:
Seems so based on mypy CI results, added it back.